test_marvin_actions.py

```diff
@@ -166,6 +166,13 @@ def testVideoOfToday(self):
                 self.assertActionOutput(marvin_actions.marvinVideoOfToday, "dagens video", response)
         self.assertActionSilent(marvin_actions.marvinVideoOfToday, "videoidag")
 
+    def testVideoOfTodayFail(self):
+        """Tests that marvin says the right thing when he doesn't have a video for the day"""
+        with mock.patch("marvin_actions.getString") as gs:
+            gs.return_value = ""
+            response = " Jag har ännu ingen passande video för denna dagen."
```

Comment:
should the leading space really be there?

Reply:
The returned string has a leading space, for some reason.
```
def videoOfToday():
    """
    Check what day it is and provide a url to a suitable video together with a greeting.
    """
    dayNum = datetime.date.weekday(datetime.date.today()) + 1
    msg = getString("weekdays", str(dayNum))
    video = getString("video-of-today", str(dayNum))

    if video:
        msg += " En passande video är " + video
    else:
        msg += " Jag har ännu ingen passande video för denna dagen."

    return msg
   ```

Reply:
maybe we should remove that in a separate PR then, leave it for now.